src/IntercomContacts.php

```diff
@@ -57,7 +62,8 @@ public function getContacts(array $options = [])
      * @return stdClass
      * @throws Exception
      */
-    public function getContact(string $id, array $options = [])
+
+    public function getContact($id, $options = [])
```

Comment:
Why are we removing the strong typing here?

Reply:
My bad. I think when I merged the change my version was included which wasn't typed. Fixed.